crt_portal/static/js/complaint_view_filters.js

```diff
@@ -53,7 +53,6 @@
    */
   function makeQueryParams(params) {
     var keys = Object.keys(params);
-
```

Comment:
This needs to be merged with Haseeb's and Hari's changes. beware of merge conflicts